src/tests/ftest/util/ddb_utils_base.py

```diff
@@ -5,21 +5,30 @@
 """
 from command_utils_base import PositionalParameter
 from command_utils import ExecutableCommand
+from general_utils import run_pcmd
 
 
 class DdbCommandBase(ExecutableCommand):
```

Comment:
This should inherit `CommandWithParameters` as there are a few methods in `ExecutableCommand` that don't make sense with the overwritten `run()` method.
```suggestion
class DdbCommandBase(CommandWithParameters):
```

This means you'll want to either replicate the `ExecutableCommand.__str__()` method in this class or move `self.sudo` and `def __str__()` to the `CommandWithParameters` class.

Reply:
To simplify our util files, would you be opposed to defining this class inside the `src/tests/ftest/util/ddb_utils.py` file instead of having an additional file?  We do this in ior_utils.py with the `IorCommand` and `Ior` classes.

Reply:
Done. I moved `DdbCommandBase` to ddb_utils.py.